lib/config.module.ts

```diff
@@ -38,39 +38,40 @@ export class ConfigModule {
    */
   static forRoot(options: ConfigModuleOptions = {}): DynamicModule {
     let validatedEnvConfig: Record<string, any> | undefined = undefined;
-    if (!options.ignoreEnvFile) {
-      if (options.validationSchema) {
-        let config = this.loadEnvFile(options);
-        if (!options.ignoreEnvVars) {
-          config = {
-            ...process.env,
-            ...config,
-          };
-        }
-        const validationOptions = this.getSchemaValidationOptions(options);
-        const {
-          error,
-          value: validatedConfig,
-        } = options.validationSchema.validate(config, validationOptions);
-
-        if (error) {
-          throw new Error(`Config validation error: ${error.message}`);
-        }
-        validatedEnvConfig = validatedConfig;
-        this.assignVariablesToProcess(validatedConfig);
-      } else {
-        const config = this.loadEnvFile(options);
-        this.assignVariablesToProcess(config);
+
+    let config = this.loadEnvFile(options);
```

Comment:
Why are we calling the `loadEnvFile()` method even if `options. ignoreEnvVars` is set to `true`?

Reply:
I can see the condition in the `loadEnvFile()` method now. Anyways, wouldn't it be cleaner to do something like this:

```suggestion
    let config = options.ignoreEnvVars ? {} : this.loadEnvFile(options);
```

instead? + removing the condition from the `loadEnvFile`